gridpapers.dtx

```diff
@@ -519,7 +518,7 @@
   },
   code={
     %% TODO Make the dashing an option
-    \tikzset{lightlines/.style={line width=0.4pt,dash=on 0.05cm off 0.05cm phase 0.025cm}}
+    \tikzset{lightlines/.style={line width=0.4pt,dash pattern=on 0.05cm off 0.05cm,dash phase=0.025cm}}
```

Comment:
I guess this is the change to make it backwards compatible? Is it the case that `dash pattern` and `dash phase` work both for older tikz and newer tikz?

Reply:
Yes, the idea is to make it backwards compatible. I think it works on newer tikz; the extent of my testing was uploading the modified .sty to overleaf, since I didn't have a newer TeXLive handy. It seemed to work there but I'm open to being proved wrong...

Reply:
It's in the latest manual, so LGTM!